Sources/ArgumentParser/Usage/UsageGenerator.swift

```diff
@@ -37,10 +37,10 @@ extension UsageGenerator {
   /// The tool synopsis.
   ///
   /// In `roff`.
-  var synopsis: String {
+  func synopsis(visibility: ArgumentVisibility = .default) -> String {
```

Comment:
Not defaulting this argument will improve correctness at the cost of verbosity. (I think its a worthwhile trade off to remove the default)